posthog/tasks/utils.py

```diff
@@ -29,8 +29,7 @@ class CeleryQueue(Enum):
     DEFAULT = "celery"
     STATS = "stats"
     EMAIL = "email"
-    INSIGHT_EXPORT = "insight_export"
-    INSIGHT_REFRESH = "insight_refresh"
+    LONG_RUNNING = "long_running"  # any task that has a good chance of taking more than a few seconds should go here
```

Comment:
Bear in mind that with this change any tasks queued in the old queues wont get consumed (at least if you also rename it in the chart)

Reply:
I don't think that those queues are used at all anymore - I can't find any (other) references to them in the code.